caregiver_application/cdr/cdr_fhir.py

```diff
@@ -285,15 +286,24 @@ def get_patient(self, patient_id):
         except:
             return None
 
-    def get_observation(self, obsv_id):
-        try:
-            if not obsv_id.startswith("Observation/"):
-                obsv_id = "Observation/" + obsv_id
-            response = requests.get(f"{self.base_url}/{obsv_id}", headers=self.header)
-            return json.loads(response.text) if response else None
-        except:
-            return None
-
+    # def get_observation(self, obsv_id):

```

Comment:
The get_observation is a well defined function on the resource to get a specific resource based on ID. It is not smart top remove that from a library (cdr looks like lib)

Reply:
I specifically added a script to handle the effective date from database, while the existing code as it is.